include/bbp/sonata/reports.h

```diff
@@ -0,0 +1,142 @@
+#pragma once
+
+/**
+ * Provides a bridge between c-based programs in order to have access to the c++-based objects
+ */
+#ifndef __cplusplus
+#include <stddef.h>
+#endif
+#if defined(__cplusplus)
+#include <cstddef>
+#include <stdint.h>
+extern "C" {
+#endif
+
+
+/*! \brief Add node to existing or new report
```

Comment:
that seems to be outdated as there is no gid or node identifier in the API

Reply:
Yes, this is out to date.